src/AuthModule/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,196 @@
+import { Toast } from "../../../Context/Components/TousterContext";
+type FormValues = {
+  oldPassword: string;
+  newPassword: string;
+  confirmPassword: string;
+};
+type Props ={
+  handleCloseDialog : ()=>void
+}
+export default function ChangePassword({ handleCloseDialog }: Props) {
+  // Inside your component function
+  const authContextValue = useContext(AuthContext); // Get the context value
+  const baseUrl: string | undefined = authContextValue
+    ? authContextValue.baseUrl
+    : undefined;
+const {showSuccessToast , showErrorToast}= useContext(Toast)
+const [spinner, setSpinner] = useState<boolean>(false);
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+    reset,
+  } = useForm<FormValues>();
+  // for showing current password
+  const [showPassword, setShowPassword] = useState(false);
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+  // for showing New password
+  const [showNewPassword, setShowNewPassword] = useState(false);
+  const handleClickShowNewPassword = () => {
+    setShowNewPassword(!showNewPassword);
+  };
+  // for showing confirm New password
+  const [showConfirmNewPassword, setShowConfirmNewPassword] = useState(false);
+  const handleClickShowConfirmNewPassword = () => {
+    setShowConfirmNewPassword(!showConfirmNewPassword);
+  };
+  // Custom validation function to check if passwords match
+  const validatePasswordMatch = (value: unknown) => {
+    const password = watch("newPassword"); // Get the value of the "password" field
+    return value === password ? true : "Passwords do not match"; // Return error message if passwords don't match
+  };
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    // Handle password change logic here
+    // const token = localStorage.getItem("userToken");
+    // if (!token) {
+    //   throw new Error("User is not authenticated");
+    // }
+    setSpinner(true)
+    const token ="Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI2NjEzZWZlZTZlYmJiZWZiYzE5ZjAwZmYiLCJyb2xlIjoiYWRtaW4iLCJ2ZXJpZmllZCI6ZmFsc2UsImlhdCI6MTcxMjU4MjY4NCwiZXhwIjoxNzEzNzkyMjg0fQ.M887Vp9SsT7OzO_fbw0acuL1PzADMB4v03Tf0DdD9Vo"
+    try {
+      const response = await axios.post(
+        `${baseUrl}/v0/admin/users/change-password`,
+        data,
+        {
+          headers: {
+            Authorization: token,
+          },
+        }
+      );
+      console.log(response.data.message);
+      showSuccessToast?.("Password has been updated successfully");
+      handleCloseDialog()
+    } catch (error) {
+      console.log(error);
+      if (axios.isAxiosError(error) && error.response) {
+        showErrorToast(error.response.data.message);
+      }
+    }finally{
+      setSpinner(false)
+    
+    }
+    reset();
+  };
+  return (
+    <form onSubmit={handleSubmit(onSubmit)}>
+      <FormControl sx={{ display: "block" }}>
+        <FormLabel sx={{ color: "black" }}>Current Password</FormLabel>
```

Comment:
We need to stick to the theme pallet for better consistence, and it will also make it easier to integrate dark/light mode functionality in our application.